Replicator/Worker.cc

```diff
@@ -71,7 +71,8 @@ namespace litecore::repl {
             } else {
                 firstline = false;
             }
-            s << format(string(kCollectionLogFormat), i++) << " "
+            s << "{Coll#" << i++ << "}"
```

Comment:
Is this correct?  Using the curly brackets is ok or is this falling back to the old way by mistake?

Reply:
This is inside a JSON representing the option, not at the level of log. I meant just to keep it that way, so that it won't be grepped mistakenly by "key=valye" pattern.